src/stories/Alert/Alerts.stories.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { AlertComponent } from './Alerts';
+import { StyledContainer } from './Alerts.style';
+export default {
+  title: 'Alert',
+  component: AlertComponent,
+};
+
+export const SimpleAlerts = () => (
+  <div>
+    <AlertComponent
+      type="primary"
+      variant="contained"
+      text="Primary! This is a Primary Alert - Check it now!"
+      disableHover
+    />
+    <AlertComponent
+      type="secondary"
+      variant="contained"
+      text="Secondary! This is a Secondary Alert - Check it now!"
+      disableHover
+    />
+    <AlertComponent
+      type="success"
+      variant="contained"
+      text="Success! This is a Success Alert - Check it now!"
+      disableHover
+    />
+    <AlertComponent
+      type="warning"
+      variant="contained"
+      text="Warning! This is a Warning Alert - Check it now!"
+      disableHover
+    />
+  </div>
+);
+
+export const OutlineAlerts = () => (
+  <StyledContainer>
+    <AlertComponent
+      type="primary"
+      variant="outline"
+      text="Primary! This is a Primary Alert - Check it now!"
+      isOutline
+      disableHover
+    />
+    <AlertComponent
+      type="textcolorgray"
```

Comment:
@Arbina001 give proper color name in theme .rename this `textcolorgray `in theme